controllers/suite_test.go

```diff
@@ -105,6 +120,13 @@ var _ = BeforeSuite(func() {
 	}).SetupWithManager(k8sManager)
 	Expect(err).ToNot(HaveOccurred())
 
+	err = (&DevfileRegistryReconciler{
+		Client: k8sManager.GetClient(),
+		Log:    ctrl.Log.WithName("controllers").WithName("DevfileRegistryReconciler"),
```

Comment:
I'm not sure if this is causing the errors in the tests, but this should be the name of the CR:
```suggestion
		Log:    ctrl.Log.WithName("controllers").WithName("DevfileRegistry"),
```

Reply:
Actually, I need to roll back these changes because the DevfileRegistry is properly tested in the integration-tests.  I'll put this PR back in WIP until I get the integration-tests updated